### ISSUE TITLE
Submit/Success Button

### ISSUE THREAD

**Issue by mgifford** [pol-is/issues#15]
A blind user will have no notice that their form is successfully submitted.

By using WAI-ARIA it is possible to alert a screen-reader user that content has changed:

<img width="837" alt="screen shot 2016-12-05 at 2 25 10 pm" src="https://cloud.githubusercontent.com/assets/116832/20899041/c0202696-baf6-11e6-8978-eab6e06f0794.png">

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

**Comment by colinmegill** [comment]
@mgifford I wouldn't want to stop you as this is, over time, going to be something we tackle. Our main blocker, however, is the participation view (as it's significantly less likely to have to accommodate a blind government employee / moderator than a blind citizen participant) visible here: mydem0cracy.ca

Would you mind starting there, and then continuing onto the backend? Having you do this pass on the participation experience would be extremely helpful. Thanks so much for your energy on a11y!

**Comment by mgifford** [comment]
Agreed that this would have a bigger impact..  I just started where I was.

**Comment by colinmegill** [comment]
:) Thanks @mgifford!
